cmd/celestia-appd/cmd/root.go

```diff
@@ -140,6 +140,7 @@ func initRootCmd(rootCmd *cobra.Command, encodingConfig encoding.Config) {
 		commands.CompactGoLevelDBCmd,
 		addrbookCommand(),
 		downloadGenesisCommand(),
+		AddrConversionCmd(),
```

Comment:
Does it make more sense to put this under the debug sub command where the other conversion like commands are based.

Reply:
Don't worry if this is too difficult to do